README.md

```diff
@@ -1,13 +1,14 @@
 # Project Name
-
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Guess who project.
+Pretty selfexplainatory. What i loved about this project was that it made me use a lot of creative tools. 
```

Comment:
It´s great to have an informative Readme file, if a tech recruiter happens to stop by and view your code. 
Maybe write that we had one week for the assignment and what kind of starter code we were given.